src/styles/index.scss

```diff
@@ -1,3 +1,287 @@
-body {
+// #region card mixin
+$primary-color: #00acdc;
+$secondary-color: #616070;
+$triple-color: #060b35;
+$four-color: #fff;
+$font-size-sm: 10px;
+$font-size-md: 12px;
+$font-size-lg: 16px;
+
+// #endregion
+// #region header
+:root {
+  --bg-color: #00acdc;
+  --count-size-monitor: 1;
+}
+
+* {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+
+  &__logo {
+    height: 60px;
+  }
+
+  &__logo-moyo {
+    margin-top: 10px;
+    margin-left: 50px;
+    height: 40px;
+  }
+}
+
+.nav {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  height: 60px;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    flex-wrap: nowrap;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+    padding: 0;
+
+    &-button {
+      display: inline-block;
+      text-align: center;
+      text-decoration: none;
+      color: #060b35;
+      height: 100%;
+      line-height: 60px;
+      position: relative;
+      cursor: pointer;
+    }
+
+    &-button:hover,
+    .is-active {
+      color: var(--bg-color);
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--bg-color);
+  border-radius: 8px;
+  opacity: 1;
+}
+
+// #endregion
+// #region main
+.main {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+
+  &__container {
+    display: grid;
+    grid-template-columns: repeat(var(--count-size-monitor), 200px);
+
+    grid-template-rows: repeat(406px);
```

Comment:
The 'grid-template-rows' property is incorrectly defined as 'repeat(406px)'. This should specify the number of rows and their size, e.g., 'repeat(2, 406px)'.